libV2/utils/disk.js

```diff
@@ -14,6 +12,12 @@ const suffixToExp = {
     z: 7,
 };
 
+async function getFolderSizeWrapper(...args) {
+    const { default: getFolderSize } = await import('get-folder-size');
+    const result = await getFolderSize(...args);
+    return result.size || 0;
+}
```

Comment:
I'm not sure I understand this one: why do we need to dynamically import this dependency? If that's needed, it's worth a comment